tests/installation/installer_desktopselection.pm

```diff
@@ -6,7 +6,7 @@ use testapi;
 sub run() {
     my %desktopkeys = (kde => "k", gnome => "g", xfce => "x", lxde => "l", minimalx => "m", textmode => "i");
     assert_screen "desktop-selection", 30;
-    my $d   = get_var("DESKTOP");
+    my $d = get_var("DESKTOP") != "awesome" ? get_var("DESKTOP") : "minimalx";
```

Comment:
And what about `!check_var('DESKTOP', 'awesome') ? get_var("DESKTOP") : "minimalx";`